docs/source/use_cases/discover_public_data/1000_genomes.rst

```diff
@@ -54,9 +54,9 @@ Google Cloud Platform data locations
 
 * Google Cloud Storage folders
    * These files were loaded into Google Genomics datasets:
-      * `gs://genomics-public-data/1000-genomes <https://console.developers.google.com/storage/genomics-public-data/1000-genomes/>`_
-      * `gs://genomics-public-data/1000-genomes-phase-3 <https://console.developers.google.com/storage/genomics-public-data/1000-genomes-phase-3/>`_
-   * A full mirror of http://ftp-trace.ncbi.nih.gov/1000genomes/ftp/ `gs://genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/ <https://console.developers.google.com/storage/browser/genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/>`_.
+      * `gs://genomics-public-data/1000-genomes <https://console.cloud.google.com/storage/genomics-public-data/1000-genomes/>`_
+      * `gs://genomics-public-data/1000-genomes-phase-3 <https://console.cloud.google.com/storage/genomics-public-data/1000-genomes-phase-3/>`_
```

Comment:
The following file still points to `developers`:

https://console.cloud.google.com/m/cloudstorage/b/genomics-public-data/o/1000-genomes/SEE_ALSO_THE_FULL_MIRROR_OF_1%2C000_GENOMES

When one clicks on it, it will show the following:

```
For more information on the data in this bucket see:
http://cloud.google.com/genomics/public-data

Note that we have a full mirror of 1,000 Genomes in Google Cloud Storage underneath this path:

  https://console.developers.google.com/storage/browser/genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp

  gs://genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp

```